apps/site/src/app/(main)/apply/sections/Form/ProfileInformation.tsx

```diff
@@ -33,26 +31,9 @@ export default function ProfileInformation() {
 					placeholder="https://"
 				/>
 			</div>
-
-			<Textfield
-				name="frq_collaboration"
-				labelClass={`${styles.label} mt-7`}
-				labelText="Why is collaboration important to being a programmer or technologist, and what does it mean to you? (150 words)"
-				inputClass={`bg-[#E1E1E1] p-3 h-48 resize-none rounded-xl`}
-				containerClass="flex flex-col w-full"
-				isRequired={true}
-				maxLength={FRQ_MAX_LENGTH}
-			/>
-
-			<Textfield
-				name="frq_dream_job"
-				labelClass={`${styles.label} mt-7`}
-				labelText="If you could have any job in the world, what would it be? (ex. YouTuber, Body Builder, etc.) (100 words)"
-				inputClass={`bg-[#E1E1E1] p-3 h-48 resize-none rounded-xl`}
-				containerClass="flex flex-col w-full"
-				isRequired={true}
-				maxLength={FRQ_MAX_LENGTH}
-			/>
+			{/*
+			TODO Add SAQs here using Textfield
+			*/}
```

Comment:
Question: is this not finished?

Reply:
this file is meant to be a template for building out each application type's unique ProfileInformation component, as each application type has differing short answer questions and essays